File: app/src/main/java/com/mjr/twaire/app/alleasy/fragment/physic/kinematics/vum/VUMSpeed4.java
package com.mjr.twaire.app.alleasy.fragment.physic.kinematics.vum;

import com.mjr.twaire.app.alleasy.fragment.physic.PhysicCalculations;
import com.mjr.twaire.app.alleasy.model.Data;

public class VUMSpeed4 extends PhysicCalculations {
    @Override
    protected void setTemplateAttributes() {
        getDatas().add(new Data("vi = ", "m/s"));
        getDatas().add(new Data("a = ", "m/s²"));
        getDatas().add(new Data("∆t = ", "s"));
    }
    
    @Override
    public String getFragmentTag() {
        return getClass().getPackage().getName() + getClass().getName();
    }
}

File: app/src/main/java/com/mjr/app/alleasy/fragment/physics/kinematics/vum/VUMSpeed4.kt
package com.mjr.app.alleasy.fragment.physics.kinematics.vum

import com.mjr.app.alleasy.activity.AbstractPhysicalCalculation
import com.mjr.app.alleasy.model.Data

class VUMSpeed4 : AbstractPhysicalCalculation() {
    override fun setTemplateAttributes() {
        datas.apply {
            add(Data("vi = ", "m/s"))
            add(Data("a = ", "m/s²"))
            add(Data("∆t = ", "s"))
        }
    }

    override fun onCalculateClickListener() {}
}